### ISSUE TITLE
ListView drag handles show despite list item hidden

### ISSUE THREAD

**Issue by max-h-silverman** [FlutterFlow/flutterflow-issues#4804]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

Handles show for hidden item

### Expected Behavior

Handles should be hidden for hidden items

### Steps to Reproduce

1. Login with flutterflow/flutterflow
2. Go to inventory screen (nav bar 3rd option)
3. Click 'show low stock only'
4. Extra set of reorder handles is visible

### Reproducible from Blank

- [ ] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT4ki87lx4lgoddA1rrtbcExvCo4H245TL07r+5xb0gvIrbyOu9/e87eXElDQtDgT1NhHFaGnnox/+jOktmWKvEqZBSqbKZqwbh2Zgr/QXq8R8SYPKWeQUF5IJ5mJnK536awkxVRAPBYWkIR6ma2H+6UVhbsGZO/Zwh9f6PHaOI=

### Visual documentation

<img width="368" alt="Screenshot 2024-11-01 at 2 15 57 PM" src="https://github.com/user-attachments/assets/fc280774-16e2-4cf7-bc20-7d1c52563c60">

### Environment

```markdown
- FlutterFlow version: 5.0.9
- Platform: macOS sequoia
- Browser name and version: chrome latest
- Operating system and version affected: osx 15.0.1
```

**Comment by Alezanello** [comment]
Hello!

I was about to download the code for your project to run it locally and test it, but I encountered some errors in the AppState that are preventing the download from completing properly. Could you try replicating this issue in a new blank project and share it with me? This way, I can test it without the errors while you are developing

Thank you!

**Comment by max-h-silverman** [comment]
@Alezanello What errors are you encountering. I'm able to download the project without any issues. I was working on something just now and maybe a temporary error happened. Please try again.

**Comment by max-h-silverman** [comment]
@Alezanello Why is this being marked stale? I'm waiting on a response.

**Comment by Alezanello** [comment]
Hello!

I’m still unable to replicate this issue on my end. To help you better, I’ve created a public project that demonstrates how I’ve set it up. You can clone this project and use it as a template to replicate each section.

In this example, I used a `DataType List` that hides items when a specific attribute is set to `false`. Please let me know if you need further clarification or run into any issues!<br><br><br>[https://app.flutterflow.io/project/qa-2953-g5j1qk](https://app.flutterflow.io/project/qa-2953-g5j1qk)

<img src="https://uploads.linear.app/3241089d-736a-4422-900b-051e72accdbe/c92ca1a7-ece2-4d56-bf3b-bb6074e321eb/7fbb88df-bce5-4b39-a0e0-7e2fb9a29149?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiLzMyNDEwODlkLTczNmEtNDQyMi05MDBiLTA1MWU3MmFjY2RiZS9jOTJjYTFhNy1lY2UyLTRkNTYtYmYzYi1iYjYwNzRlMzIxZWIvN2ZiYjg4ZGYtYmNlNS00YjM5LWEwZTAtN2UyZmI5YTI5MTQ5IiwiaWF0IjoxNzMyMTk5MjY1LCJleHAiOjMzMzAyNzU5MjY1fQ.A75A52w_-NijMd0-t59fO1Cr8MCdT_h-wV-VVMusEY0 " alt="listReorder.gif" width="1420" height="888" />

**Comment by Alezanello** [comment]
Closing this issue since we haven't heard back from you. If you are still facing the problem please submit a new issue. Have a great day!
